Neos.Neos/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -107,6 +107,10 @@ additionalProperties:
 
                       'placeholder': { type: string, description: "Optional text that will be rendered in the aloha editor when there is no content." }
 
+                      'sampleCase' : { type: any }
+
+                      'fallbackCase' :  { type: any }
```

Comment:
True ... but they have to be there to have a valid configuration. Otherwise we cannot test the Testing configuration